auth/README.md

```diff
@@ -134,8 +135,17 @@ docker service create --name auth \
  -e cookie_root_domain=".system.gw.io" \
  -e public_key_path=/run/secrets/jwt-public-key \
  -e private_key_path=/run/secrets/jwt-private-key \
+ -e oauth_provider="github" \
  --secret jwt-private-key \
  --secret jwt-public-key \
  --secret of-client-secret \
  openfaas/cloud-auth:$TAG
 ```
+
+### GitLab integration
+
+If you want to integrate OpenFaaS Cloud with your self-managed GitLab you need to set env variables:
+```
+oauth_provider="gitlab"
+oauth_provider_base_url="..." # where instead of ... you should put valid url to your self-manager GitLab (for example: https://gitlab.domain.com)
```

Comment:
Instead of a long comment add this above the block. "self-manager" is a typo. It would be "self-managed" or better "self-hosted".

Reply:
Changed